src/storage/txn/actions/acquire_pessimistic_lock.rs

```diff
@@ -36,11 +36,24 @@ pub fn acquire_pessimistic_lock<S: Snapshot>(
     need_check_existence: bool,
     min_commit_ts: TimeStamp,
     need_old_value: bool,
+    lock_only_if_exists: bool,
 ) -> MvccResult<(Option<Value>, OldValue)> {
     fail_point!("acquire_pessimistic_lock", |err| Err(
         crate::storage::mvcc::txn::make_txn_error(err, &key, reader.start_ts).into()
     ));
-
+    if lock_only_if_exists && !need_value {
+        error!(
+            "When acquires a pessimistic lock, Lock_only_if_exists of a pessimistic lock request is set to true,
+            but return_value is not";
```

Comment:
```suggestion
            "lock_only_if_exists of a pessimistic lock request is set, but return_value is not";
```

It looks not fit in our style of logs.